main.py

```diff
@@ -232,6 +233,15 @@ def create_circuit(self, request: Request) -> JSONResponse:
             log.debug("create_circuit result %s %s", exception, 400)
             raise HTTPException(400, detail=str(exception)) from exception
 
+        if evc.is_intra_switch():
+            try:
+                self.check_disabled_component(evc.uni_a, evc.uni_z)
+            except HTTPException as exception:
+                raise HTTPException(
+                    409,
+                    detail=f"Path is not valid: {exception}"
```

Comment:
The current `exception` str repr is being a bit vague:
```
{
    "description": "Path is not valid: 409",
    "code": 409
}
```

Either you can refactor `check_disabled_component` to raise `napps.kytos.mef_eline.exceptions.InvalidPath`, or pass here `{exception.detail}`. I think raising `InvalidPath` would be a bit cleaner since that one is just business logic and you're already handling the inner exception and re-raising as `HTTPException`. Wdyt?

Reply:
Overlook this. Fixed. Checking for unis in `evc.update()`. Also I added `DisabledSwitch` to exceptions.